python/foglamp/services/core/routes.py

```diff
@@ -11,9 +11,10 @@
 from foglamp.services.core.api import scheduler as api_scheduler
 from foglamp.services.core.api import statistics as api_statistics
 from foglamp.services.core.api import backup_restore
+from foglamp.services.core.api import update
 
-__author__ = "Ashish Jabble, Praveen Garg"
-__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__author__ = "Ashish Jabble, Praveen Garg, Massimiliano Pinto"
+__copyright__ = "Copyright (c) 2017-2018 OSIsoft, LLC"
```

Comment:
not sure we want to update for single file, @izoratti can help here better.

Reply:
ok